flask/boxwise_flask/models/x_beneficiary_language.py

```diff
@@ -0,0 +1,27 @@
+from boxwise_flask.db import db
+from boxwise_flask.models.beneficiary import Beneficiary
+from boxwise_flask.models.language import Language
+from peewee import ForeignKeyField
+
+
+class XBeneficiaryLanguage(db.Model):
+    language = ForeignKeyField(
+        column_name="language_id",
+        field="id",
+        model=Language,
+        on_update="CASCADE",
```

Comment:
Do we want to cascade cross reference tables? I'm a bit rusty but I think there may be some risk there

Reply:
In `init.sql` we have
```
  KEY `language_id` (`language_id`),
  CONSTRAINT `x_people_languages_ibfk_2` FOREIGN KEY (`language_id`) REFERENCES `languages` (`id`) ON DELETE CASCADE ON UPDATE CASCADE,
```
The model is derived from those instructions. If we want to have it defined differently it should happen in the 'upstream' dropapp database, and then be ported here, I think.

Reply:
Same for the `cms_usergroups_functions` table and the `UsergroupModuleAccess` model, resp.

Reply:
Noted, logged here: https://trello.com/c/OQPbLsvo